docker/nginx/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM nginx
```

Comment:
Let's pin this explicitly to the current nginx version:

```
FROM nginx:1.11.5
```

And then we should tag the image that you build from this file as buoyantio/nginx:1.11.5.